client/common/config/config.js

```diff
@@ -1,4 +0,0 @@
-modules.define('config', function (provide) {
```

Comment:
`config` is not only about app params. It's about project params. We can use it in different blocks.